extra_data/cli/make_virtual_cxi.py

```diff
@@ -56,8 +41,14 @@ def main(argv=None):
              "By default, it is written in the proposal's scratch directory."
     )
     ap.add_argument(
-        '--min-modules', type=int, default=9, metavar='N',
-        help="Include trains where at least N modules have data (default 9)"
+        '--min-modules', type=int, default=None, metavar='N',
+        help='Include trains where at least N modules have data (default'
+             ' 9 for AGIPD1M/DSSC1M/LPD1M, 5 for AGIPD500K and 1 otherwise)'
+    )
+    ap.add_argument(
+        '--n-modules', type=int, default=None, metavar='N',
+        help='Number of detector modules in the experiment setup.'
+             ' Should be used only for Jungfrau data.'
```

Comment:
```suggestion
             ' Should be used only for JUNGFRAU data.'
```

PSI always capitalise [JUNGFRAU](https://www.psi.ch/en/detectors/jungfrau). It's a bit silly, but after spending many years working on IPython (*not* iPython, as many people write), I try to follow how people choose to write the name of their projects. :slightly_smiling_face: